templates/sysconfig.epp

```diff
@@ -15,6 +15,21 @@ COMPRESSAFTER=<%= $compressafter %>
 # which are used for the generation of log files.
 SADC_OPTIONS="<%= $sadc_options %>"
 
+# Directory where sa and sar files are saved. The directory must exist.
+SA_DIR=<%= $sa_dir %>
+
```

Comment:
But your code creates it, so maybe change the commentary to 'The directory will be created if it does not exist'.

Reply:
I was trying to preserve the comment from the "un-puppetized" code.  The sysstat tools wont make the directory if it is missing.  If someone disables puppet and changes this value, they'll need to make the directory.

Reply:
sorry, bit of a brain freeze on my part - was thinking you were updating the read me - all good